docs/content/guide/external.ngdoc

```diff
@@ -0,0 +1,146 @@
+* **Visualization:** [SVG](http://gaslight.co/blog/angular-backed-svgs), [D3.js](http://www.ng-newsletter.com/posts/d3-on-angular.html)
+* **Realtime Communication: **[Socket.io](http://www.creativebloq.com/javascript/angularjs-collaboration-board-socketio-2132885), [OmniBinder](https://github.com/jeffbcross/omnibinder)
+
+
+## Tools
+
+* **Getting Started:** [Comparison of the options for starting a new project](http://www.dancancro.com/comparison-of-angularjs-application-starters/)
+* **Debugging:** [Batarang](https://chrome.google.com/webstore/detail/angularjs-batarang/ighdmehidhipcmcojjgiloacoafjmpfk?hl=en)
+* **Editor support:** [Webstorm](http://plugins.jetbrains.com/plugin/6971) (and [video](http://www.youtube.com/watch?v=LJOyrSh1kDU)), [Sublime Text](https://github.com/angular-ui/AngularJS-sublime-package), [Visual Studio](http://madskristensen.net/post/angularjs-intellisense-in-visual-studio-2012)
+* **Workflow:** [Yeoman.io](https://github.com/yeoman/generator-angular) and [Angular Yeoman Tutorial](http://www.sitepoint.com/kickstart-your-angularjs-development-with-yeoman-grunt-and-bower/)
+
+## Complementary Libraries
+
+This is a list of libraries that enhance Angular, add common UI components or integrate with other libraries.
+You can find a full list of Angular external libraries at [ngmodules.org](http://ngmodules.org/).
+
+* **Advanced Routing:** [UI-Router](https://github.com/angular-ui/ui-router)
+* **Authentication:** [Http Auth Interceptor](https://github.com/witoldsz/angular-http-auth)
+* **Internationalization:**
+  - [angular-translate](http://angular-translate.github.io)
+  - [angular-gettext](http://angular-gettext.rocketeer.be/)
+  - [angular-localization](http://doshprompt.github.io/angular-localization/)
+* **RESTful services:** [Restangular](https://github.com/mgonto/restangular)
+* **SQL and NoSQL backends:**
+  - [BreezeJS](http://www.breezejs.com/)
+  - [AngularFire](http://angularfire.com/)
+* **Data Handling**
+  - Local Storage and session: [ngStorage](https://github.com/gsklee/ngStorage)
+  - [angular-cache](https://github.com/jmdobry/angular-cache)
+  - Data Modeling [JS-Data-Angular](https://github.com/js-data/js-data-angular)
+* **Fileupload:**
+  - [ng-file-upload](https://github.com/danialfarid/ng-file-upload)
+  - [blueimp-fileupload for Angular](https://blueimp.github.io/jQuery-File-Upload/angularjs.html)
+* **Specific UI Elements:**
+  - [ngInfiniteScroll](https://sroze.github.io/ngInfiniteScroll/)
+  - [ngTable](https://github.com/esvit/ng-table)
+  - [Angular UI Grid](http://angular-ui.github.io/grid)
+  - [Toaster Notifications](https://github.com/jirikavi/AngularJS-Toaster)
+  - [textAngular Rich Text Editor / contenteditable](http://textangular.com/) (Rich Text Editor /
+    binding to contenteditable)
+  - [Angular UI Map (Google Maps)](https://github.com/angular-ui/ui-map)
+* **General UI Libraries:**
+  - [AngularStrap for Bootstrap 3](http://mgcrea.github.io/angular-strap/)
```

Comment:
This should probably be moved together with UI Bootstrap (since they are similar) and let `ngMaterial` make it to the top :wink: